### DISCUSSION TITLE
SEQ server fails to start when running on k8s with an existing PVC (NFS)

### DISCUSSION THREAD

**Discussion by Talha-Altair** [datalust/seq-tickets#1846] [Help] [answered]
SEQ server fails to start when running on k8s with an existing PVC (NFS):

I am running an on-prem server, which has Kubernetes running and using rancher as the dashboard tool.

I have a NFS server running on the same network and am able to create PVC out of the NFS server and use it for other workloads.

For SEQ in particular, this is the error i am faced with;

![image](https://github.com/datalust/seq-tickets/assets/70721398/8d339b83-bd21-4850-8263-711245dbcbfb)

**Comment by liammclennan** [comment]
Hi @Talha-Altair ,

See https://docs.datalust.co/docs/kubernetes-storage

When Seq starts it attempts to lock its storage, hence only one Seq process can run against a particular file system location at one time. Seq is failing to start because something (most likely another instance of Seq) has already locked the files. Another possibility is that a virus scanner could be locking the files, so best to excludes Seq's storage directory from antivirus scanning.

**Reply by Talha-Altair** [reply]
Thanks for the reply, but, it's a new folder created , and no other instance of seq is running in the nas server. My suspicion is that, NFS maybe blocking seq storage lock.

**Reply by liammclennan** [reply, marked as answer]
Many network file systems do not provide the reliability needed to run Seq (or any database).

**Reply by dheerajd3v** [reply]
@liammclennan We are seeing same issue with SEQ using Longhorn storage
